examples/textarea/src/main.rs

```diff
@@ -0,0 +1,120 @@
+use std::{cell::RefCell, io, rc::Rc};
+
+use ratatui::prelude::Stylize;
+use ratzilla::{
+    event::{KeyCode, KeyEvent},
+    DomBackend, WebRenderer,
+};
+
+fn main() -> io::Result<()> {
+    std::panic::set_hook(Box::new(console_error_panic_hook::hook));
+
+    let backend = DomBackend::new()?;
+    let terminal = ratatui::Terminal::new(backend)?;
+
+    let app = Rc::new(RefCell::new(App::new()));
+
+    terminal.on_key_event({
+        let event_state = app.clone();
+        move |key_event| {
+            let mut state = event_state.borrow_mut();
+            state.handle_events(key_event);
+        }
+    });
+
+    terminal.draw_web({
+        let render_state = app.clone();
+        move |frame| {
+            let state = render_state.borrow();
+            state.render(frame);
+        }
+    });
+
+    Ok(())
+}
+
+#[derive(Default)]
+struct App<'a> {
+    textarea: tui_textarea::TextArea<'a>,
+    status_text: String,
+}
+
+impl<'a> App<'a> {
+    fn new() -> Self {
+        let mut textarea = tui_textarea::TextArea::default();
+        textarea.set_block(
+            ratatui::widgets::Block::default()
+                .borders(ratatui::widgets::Borders::ALL)
+                .title("Minimal Textarea Example"),
+        );
+
+        App {
+            textarea,
+            status_text: String::new(),
+        }
+    }
+
+    fn render(&self, frame: &mut ratatui::Frame) {
+        let chunks = ratatui::layout::Layout::vertical([
+            ratatui::layout::Constraint::Min(0),
+            ratatui::layout::Constraint::Length(3),
+        ])
+        .split(frame.area());
+
+        let style = ratatui::style::Style::new().cyan().italic();
+        let status = ratatui::text::Span::styled(self.status_text.as_str(), style);
```

Comment:
Can you use implicit imports (from `ratzilla::ratatui`)?

```suggestion
        let style = Style::new().cyan().italic();
        let status = Span::styled(self.status_text.as_str(), style);
```

We need to update the other occurrences as well :)